docs/docs/deploying-to-now.md

```diff
@@ -2,36 +2,62 @@
 title: Deploying to Now
 ---
 
-In order to deploy your Gatsby project using [Now](https://zeit.co/now), you can do the following:
+[ZEIT Now](https://zeit.co/now) is a [cloud platform for serverless deployment](https://zeit.co/docs/v2/getting-started/introduction-to-now/) that you can use to deploy your Gatsby projects and [alias them](https://zeit.co/docs/v2/domains-and-aliases/aliasing-a-deployment/) to your domain or a free `.now.sh` suffixed URL.
 
-1.  Install the Now CLI
+This guide will show you how to get started in a few quick steps:
 
-`npm install -g now`
+## Step 1: Getting Started with Gatsby
+If you haven't already [set up a Gatsby project](https://www.gatsbyjs.org/docs/quick-start) you can do so by using npm's `npx` command in your terminal:
 
-2.  Run `gatsby build` at the root of your project.
+```shell
+npx gatsby new <your project name>
+```
 
-3.  Run `now` inside `public/`. This will upload your project to the cloud.
+## Step 2: Getting Now
 
-For large project sizes, it is better to install a Node server package (such as `serve`, or `http-server`):
+If you're not acquainted with Now, it is an easy-to-use cloud platform for serverless deployments that you can use from [the command-line](https://zeit.co/docs/v2/getting-started/installation/#now-cli) or [the desktop app](https://zeit.co/docs/v2/getting-started/installation/#now-desktop)
```

Comment:
This can be simplified since you already explained what Now is at the beginning of the guide